### PR TITLE
fix(es/resolver): Ignore qualifiers of `TsImportType`

### PR DESCRIPTION
**Pull request by dsherret**
**Description:** The `Resolver` was resolving the qualifier of `TsImportType`

**Related issue (if exists):** None

### CODE REVIEW

File: crates/swc_ecma_transforms_base/tests/ts-resolver/ts_import_type_qualifier/output.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { MyClass__2 } from "./MyClass.ts";
+const test__2 = new MyClass__2();
+export type MyType__2 = import("./deps.ts").MyClass__0;
```
Comment (dsherret) [inline crates/swc_ecma_transforms_base/tests/ts-resolver/ts_import_type_qualifier/output.ts:3]: This was previously:

```
export type MyType__2 = import("./deps.ts").MyClass__2;
```

**Comment by dsherret** [comment]
CI failures seem unrelated.

**Review by kdy1 — APPROVED** [review]
swc-bump:
 - swc_ecma_transforms_base

**Comment by dsherret** [comment]
CI failures seem due to https://github.com/actions/setup-node/issues/899

**Pull request merged**